arviz/plots/plot_utils.py

```diff
@@ -242,6 +242,25 @@ def xarray_var_iter(data, var_names=None, combined=False, skip_dims=None, revers
         The string is the variable name, the dictionary are coordinate names to values,
         and the array are the values of the variable at those coordinates.
     """
+
+    def ordered_unique(l):
+        """Remove duplicates from list while conserving order
+        Required arguments:
+            l: Iterable
+        Returns:
+            First occurences in order
+        """
+    
+        def purge_duplicates(l):
+            seen = set() # for faster lookups than lists
+            for x in l:
+                if x in seen:
+                    continue
+                seen.add(x)
+                yield x
+    
+        return np.array(list(purge_duplicates(l)))
```

Comment:
Could the same functionality be achieved with
```
_list = []
for item in l:
    if item not in _list:
        _list.append(item)
return _list
```

Reply:
The biggest thing is it cuts down 26 lines of code to 5 :)

Reply:
updated!